wandb/deploys/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-bullseye
+
+RUN apt-get update && apt-get upgrade && apt-get install
+# install git
+RUN apt-get install -y git
+RUN rm -rf /var/lib/apt/lists
```

Comment:
The `rm -rf` here is doing nothing.  Docker caches it's builds based on each "layer", every top level docker command like `RUN` creates a new layer.  Each layer looks at the state of the file system before it's run, then executes whatever it's meant to execute and stores a diff of the filesystem after it was run.  It's generally best practice to create the smallest number of changes to the file system as possible to reduce the overall size of the docker container.  When updating packages in ubuntu this means getting rid of the big index files it downloads to `/var/lib/apt/lists`.  Additionally it's good practice to ask apt to cleanup after itself, so the 3 lines above should be changed to:

```dockerfile
RUN apt-get update && apt-get upgrade -y \
  && apt-get install -y git \
  && apt-get -qy autoremove \
  && apt-get clean && rm -r /var/lib/apt/lists/*
```

Reply:
Okay, thanks for the info. I'll make that change now.